packages/rulesets/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft.azure/openapi-validator-rulesets",
-  "version": "1.3.1",
+  "version": "1.3.2",
```

Comment:
When updating the version make sure to use `rush version --bump` so the changelog gets generated. See https://github.com/Azure/azure-openapi-validator/blob/main/CONTRIBUTING.md#create-a-release-pr

Reply:
good point !
Ill do that, thanks

Reply:
When you do version bump, you need to discard other file changes.

Reply:
@tejaswiMinnu  wdym by other file changes?

Reply:
Few files were deleted .... [rkmanda-fixClientFlattenDocumentation_2023-12-09-02-48.json](https://github.com/Azure/azure-openapi-validator/pull/643/files#diff-812bc9dbff6f86bf7444a1b41ae539f7c6372527007e0912fcc35333b4a096e4)

Reply:
yeah, that is expected.
when we do the version bump, individual changeLog files get deleted and all those are consolidated and added to the main changelog.json